### DISCUSSION TITLE
How to use Pagination?

### DISCUSSION THREAD

**Discussion by zeshhaan** [element-plus/element-plus#6489] [Q&A]
Hi folks!
I'm having trouble with linking pagination to my form checkbox group. 
I can't limit items per active page, it populates all items
I'm a newbie, so any help is appreciated 🙏

Here is my link to the SFC playground [link](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZGl2PlxuICAgIDxlbC1jaGVja2JveC1ncm91cCB2LW1vZGVsPVwiY2hlY2tib3hHcm91cFwiIHNpemU9XCJsYXJnZVwiPlxuICAgICAgPGVsLWNoZWNrYm94LWJ1dHRvbiB2LWZvcj1cImNpdHkgaW4gY2l0aWVzXCIgOmtleT1cImNpdHlcIiA6bGFiZWw9XCJjaXR5XCI+XG4gICAgICAgIHt7IGNpdHkgfX1cbiAgICAgIDwvZWwtY2hlY2tib3gtYnV0dG9uPlxuICAgICAgPGVsLXBhZ2luYXRpb24gXG4gICAgICAgICAgICAgICAgICAgICBiYWNrZ3JvdW5kIFxuICAgICAgICAgICAgICAgICAgICAgbGF5b3V0PVwicGFnZXJcIiBcbiAgICAgICAgICAgICAgICAgICAgIDp0b3RhbD1cImNpdGllcy5sZW5ndGhcIiBcbiAgICAgICAgICAgICAgICAgICAgIDpwYWdlLXNpemU9XCI0XCJcbiAgICAgICAgICAgICAgICAgICAgID5cbiAgICAgIDwvZWwtcGFnaW5hdGlvbj5cbiAgICA8L2VsLWNoZWNrYm94LWdyb3VwPlxuICA8L2Rpdj5cblx0XG48L3RlbXBsYXRlPlxuXG48c2NyaXB0IGxhbmc9XCJ0c1wiIHNldHVwPlxuaW1wb3J0IHsgcmVmIH0gZnJvbSAndnVlJ1xuY29uc3QgY2hlY2tib3hHcm91cCA9IHJlZihbJ1NoYW5naGFpJ10pXG5jb25zdCBjaXRpZXMgPSBbJ1NoYW5naGFpJywgJ0JlaWppbmcnLCAnR3Vhbmd6aG91JywgJ1NoZW56aGVuJywgJ0thc2FyYWdvZCcsICdDTFQnLCAnU0YnLCAnTllDJywnTUFRJywgJ0RYQiddXG48L3NjcmlwdD5cblxuPHN0eWxlIHNjb3BlZD5cbi5kZW1vLWJ1dHRvbi1zdHlsZSB7XG4gIG1hcmdpbi10b3A6IDI0cHg7XG59XG48L3N0eWxlPiIsImltcG9ydC1tYXAuanNvbiI6IntcImltcG9ydHNcIjp7fX0ifQ==)

**Comment by msidolphin** [comment]
This is a very strange usage, i suggest you use select, because you have a lot of options
